storyblok/storyblok-design-system
# fix: update empty state size

## Pull request type

Jira Link: [SHAPE-2441](https://storyblok.atlassian.net/browse/SHAPE-2441)




## How to test this PR

In the visual editor, go to a story without any discussions and check if the icon size of the empty state is correct.

![image](https://github.com/storyblok/storyblok-design-system/assets/11169758/f533d646-0838-4871-a462-8a207a5945af)

Obs., this PR contains code from other PR's merged to develop without being deployed to production.

[SHAPE-2441]: https://storyblok.atlassian.net/browse/SHAPE-2441?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
